incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -32,19 +33,35 @@ var (
 )
 
 var treeCmd = &cobra.Command{
-	Use:   "tree",
-	Short: "Displays the hierarchy tree rooted at the given namespace",
-	Args:  cobra.ExactArgs(1),
+	Use:     "tree",
+	Short:   "Displays the hierarchy tree rooted at the given namespaces separated by space and if no namespace is provided then throws an error, and in case of --all-namespace/-A flag shows tree for all root namespaces",
```

Comment:
This isn't really a short description any more - I'd change this to "Display one or more hierarchy trees." If you add help text to `-A`, then saying `kubectl hnc help tree` _should_ give enough information to make it clear.